src/pages/patientView/mutation/column/CohortColumnFormatter.tsx

```diff
@@ -61,31 +62,13 @@ export default class CohortColumnFormatter {
 
     private static makeCohortFrequencyViz(variantCount:AugVariantCountOutput | null) {
 
+        let status:"loading"|"error"|"na";
         if (variantCount === null) {
-            return (
-                <span
-                    style={{color: "gray", fontSize:"xx-small", textAlign:"center"}}
-                    alt="Querying server for data."
-                >
-                    LOADING
-                </span>
-            );
+            status = "loading";
         } else if (variantCount.status === "error") {
-            return (<span
-                style={{color: "gray", fontSize:"xx-small", textAlign:"center"}}
-                alt="Error retrieving data."
-            >
-                    ERROR
-                </span>);
+            status = "error";
         } else if (variantCount.data === null) {
-            return (
-                <span
-                    style={{color: "gray", fontSize:"xx-small", textAlign:"center"}}
-                    alt="mRNA data is not available for this gene."
-                >
-                    NA
-                </span>
-            );
+            status = "na";
         } else {
```

Comment:
typescript isn't smart enough to infer the type guards on variantCount, so it complains if it's not in the attached `} else {`